### ISSUE TITLE
Combobox sets an empty aria-controls attribute when using a dialog popup

### ISSUE THREAD

**Issue by TomSchrier** [angular/components#33640]
### Which @angular/* package(s) are the source of the bug?

Don't known / other

### Is this a regression?

No

### Description

This happens in the `@angular/aria` package using the `Combobox` directives.

I am following the [*Dialog popup* example on angular.dev](https://angular.dev/guide/aria/combobox#dialog-popup) to implement a combobox with a popup containing a searchable listbox.

When the popup is opened, the outer combobox receives an `aria-controls` attribute with an empty value:

```html
<div
  role="combobox"
  aria-expanded="true"
  aria-haspopup="listbox"
  aria-controls=""
>
```

The popup itself contains a second combobox (the search input), which correctly references the listbox:

```html
<input
  role="combobox"
  aria-controls="ng-listbox-..."
/>
```

Because the outer combobox renders `aria-controls=""`, Axe reports this as an accessibility violation. [According to MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/combobox_role#:~:text=When%20a%20combobox%27s%20popup%20is%20displayed%2C%20ensure%20the%20aria%2Dcontrols%20attribute%20on%20the%20combobox%20element%20is%20set%20to%20the%20id%20of%20the%20popup%20listbox%2C%20tree%2C%20grid%2C%20or%20dialog%20element%2E) `aria-controls` should be set to the id of the popup. I assume this is the element containing the `ComboboxPopup` or `ComboboxWidget` directive.

### Please provide a link to a minimal reproduction of the bug

https://stackblitz.com/edit/stackblitz-starters-cc2eb68f

### Please provide the exception or error you saw

```true

```

### Please provide the environment you discovered this bug in (run `ng version`)

```true
Angular CLI       : 22.1.2
Angular           : 22.1.0
Node.js           : 24.16.0
Package Manager   : pnpm 11.19.0
Operating System  : darwin arm64

┌───────────────────────────────────┬───────────────────┬───────────────────┐
│ Package                           │ Installed Version │ Requested Version │
├───────────────────────────────────┼───────────────────┼───────────────────┤
│ @angular/aria                     │ 22.1.0            │ ^21.0.5           │
│ @angular/build                    │ 22.1.2            │ 22.1.2            │
│ @angular/cdk                      │ 22.1.0            │ ^21.0.5           │
│ @angular/cli                      │ 22.1.2            │ 22.1.2            │
│ @angular/common                   │ 22.1.0            │ ^22.0.0           │
│ @angular/compiler                 │ 22.1.0            │ ^22.0.0           │
│ @angular/compiler-cli             │ 22.1.0            │ 22.1.0            │
│ @angular/core                     │ 22.1.0            │ ^22.0.0           │
│ @angular/forms                    │ 22.1.0            │ ^22.0.0           │
│ @angular/platform-browser         │ 22.1.0            │ ^22.0.0           │
│ @angular/platform-browser-dynamic │ 22.1.0            │ ^22.0.0           │
│ @angular/router                   │ 22.1.0            │ ^22.0.0           │
│ rxjs                              │ 7.8.2             │ ~7.8.2            │
│ typescript                        │ 6.0.3             │ ~6.0.3            │
└───────────────────────────────────┴───────────────────┴───────────────────┘
```
